frontend/public/kubevirt/components/vm/vm-detail.jsx

```diff
@@ -30,15 +34,29 @@ import { Disk } from '../disk';
 import { NodeLink, LoadingInline } from '../../../components/utils';
 import { menuActions } from './menu-actions';
 
-const VmiEvents = ({obj: vm}) => {
-  const vmi = {
-    kind: VirtualMachineInstanceModel.kind,
-    metadata: {
-      name: vm.metadata.name,
-      namespace: vm.metadata.namespace,
-    },
+const VmEvents = ({ obj: vm }) => {
+  const vmObj = {
+    name: getName(vm),
+    namespace: getNamespace(vm),
+  };
+  const vmFilter = obj => _.isMatch(obj, {...vmObj, kind: VirtualMachineModel.kind});
+  const vmiFilter = obj => _.isMatch(obj, {...vmObj, kind: VirtualMachineInstanceModel.kind});
+  const launcherPodFilter = ({ kind, namespace, name }) =>
+    kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith(`${VIRT_LAUNCHER_POD_PREFIX}${getName(vm)}-`);
+  const importerPodFilter = ({ kind, namespace, name }) => {
+    // importer pod example importer-<diskName>-<vmname>-<generatedId>
+    // note: diskName and vmname may contain '-' which means pod name should have at least 4 parts
+    if (kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith('importer-')) {
+      const firstDashIndex = name.indexOf('-');
+      const lastDashIndex = name.lastIndexOf('-');
+      const diskAndVmName = name.slice(firstDashIndex + 1, lastDashIndex);
```

Comment:
we can also exclude the disk (or at least the first part of it). Maybe check the number of dashes first?

Reply:
added check for number of dashes and I also exclude disk (or part of it) from final check.